ml-api-adapter/chart-handler-notification/values.yaml

```diff
@@ -133,6 +133,12 @@ config:
   event_async_override: 'log,trace'
   event_traceid_per_vendor: false
 
+  # Parameters for JWS signing requests generated by ml-api-adapter
+  endpointSecurity:
+    jwsSign: false
+    fspiopSourceSigningName: 'switch'
+    jwsSigningKey:
```

Comment:
Please add some documentation here, perhaps showing an example of what one would expect the value of `jwsSigningKey` to look like.